fuse/ipns/ipns_unix.go

```diff
@@ -123,12 +123,12 @@ func CreateRoot(n *core.IpfsNode, keys []ci.PrivKey, ipfsroot string) (*Root, er
 			continue
 		}
 
-		if !u.IsValidHash(pointsTo) {
+		if !u.IsValidHash(pointsTo.B58String()) {
```

Comment:
this is encoding every time... down the road, we should cache these values. dont worry about it now